src/helpers/formatTechnologyName.js

```diff
@@ -0,0 +1,7 @@
+import formatString from 'helpers/formatString'
+
+export default (option) => {
+  const rnaSeq = 'RNA-seq' // 'rna-seq' should be rendered as 'RNA-seq' in UI
+
+  return option.match(rnaSeq.toLowerCase()) ? rnaSeq : formatString(option)
+}
```

Comment:
`technologyNames` would make sense to put in config now

```suggestion
const technologyNames = {
  'rna-seq': 'RNA-seq'
}
export default (option) => {
  return technologyNames[option.toLowerCase()] || formatString(option)
}
```